js/Ass.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function(){
+$('#first-name').text("Mahougbe");
+$('#last-name').text("Houffon");
+$('.mcc').css({"color": "blue", "font-style": "italic"});
```

Comment:
Beautiful anonymous function that sets the name text and the style of the mcc class. Your just missing those list items.